packages/web/src/components/organisms/PossessionOutline/index.tsx

```diff
@@ -41,7 +35,11 @@ export const PossessionOutline = ({ propertyAddress }: Props) => {
         <Col flex="1 1 148px">
           <Statistic
             title="Average Interest Rate"
```

Comment:
Do you want to keep the wording as it is?

Reply:
I changed the statistic title!